variants/XIAO_ESP32C6/variant.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+*By setting the WIFI_ENABLE and WIFI_ANT_CONFIG pins,
+*
+*the XIAO_ESP32C6 will turn on the on-board antenna by default after power-on
+*
+*https://wiki.seeedstudio.com/xiao_esp32c6_getting_started/
+*/
+  
+#include "esp32-hal-gpio.h"
+#include "pins_arduino.h"
+
+extern "C" {
+
+void initVariant(void) {
+  pinMode(WIFI_ENABLE, OUTPUT);
+  digitalWrite(WIFI_ENABLE, LOW);//turn on this function
+
+  pinMode(WIFI_ANT_CONFIG, OUTPUT); 
+  digitalWrite(WIFI_ANT_CONFIG, LOW);//use external antenna
```

Comment:
According to the docs for XIAO_ESP32C6 this line is not true right now:


> GPIO14 is used to select between using the built-in antenna or an external antenna. Before that, you need to set GPIO3 low level to turn on this function. If GPIO14 is set low level, it uses the built-in antenna; if it set to high level, it uses the external antenna. Default is low level. If you want to set it high, you can refer the code below.
```
void setup() {
  pinMode(3, OUTPUT);
  digitalWrite(3, LOW);//turn on this function
  delay(100);
  pinMode(14, OUTPUT); 
  digitalWrite(14, HIGH);//use external antenna
}

```
```suggestion
  digitalWrite(WIFI_ANT_CONFIG, LOW);//use built-in antenna, set HIGH to use external antenna
```